server.js

```diff
@@ -1,10 +1,25 @@
 const express = require('express');
-const bodyparser =  require ('body-parser');
+const bodyparser =  require('body-parser');
 const mongoose =  require('mongoose');
-var Book = require('./models/books');
+var Book =require('./models/books');
```

Comment:
`const Book = require('./models/books');` use const instead of var, read more on why.

Reply:
okay, I would research on that